cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -662,13 +662,22 @@ func (c serviceAccountTokenControllerStarter) startServiceAccountTokenController
 	if err != nil {
 		return nil, false, fmt.Errorf("failed to build token generator: %v", err)
 	}
+
+	legacyTokenCleanUp := utilfeature.DefaultFeatureGate.Enabled(kubefeatures.LegacyServiceAccountTokenCleanUp)
+	var secretResync time.Duration
+	if legacyTokenCleanUp {
+		secretResync = 24 * time.Hour
```

Comment:
I'm trying to remember the implications of triggering a resync in a shared informer... does this mean all secret-watching informers will get a resync triggered?

Reply:
No. Only the informer registers event handler via AddEventHandlerWithResyncPeriod function and with resyncPeriod specified will get the resync triggering.